charts/rancher-csp-adapter/0.1.0+up0.1.0-rc2/values.yaml

```diff
@@ -0,0 +1,19 @@
+debug: true
+
+image:
+  repository: rancher/rancher-csp-adapter
+  tag: v0.1.0-rc2

```

Comment:
We should move the chart and all tags to 1.0.0 on release, but we can merge this as-is to get QA kicked off.